bigbluebutton-html5/imports/ui/stylesheets/variables/palette.scss

```diff
@@ -18,3 +18,6 @@ $color-link: $color-primary !default;
 $color-link-hover: darken($color-link, 15%) !default;
 
 $color-gray-label: $color-gray !default;
+
+$user-list-bg: #F3F6F9;
+$list-item-bg-hover: darken(#F3F6F9, 7%);
```

Comment:
I don't see this variable used anywhere. But there is a duplicate under `imports/ui/stylesheets/variables/focus.scss`.
Am I missing something?